examples/ReactExample/components/todo-app.js

```diff
@@ -18,10 +18,13 @@
 
 'use strict';
 
-import React, {
+import React from 'react';
+
+import {
```

Comment:
The deprecation is that the `react-native` module no longer re-exports `React`, so you need to import it separately.